https://github.com/rack/rack-cache/pull/14

From db34e2b5ab7b16051f7158b2896e56502992c33e Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Mon, 3 Apr 2023 03:31:31 +0100
Subject: [PATCH] test: Add explicit 'timeout' require

Fixes the following when running tests individually:
```
  1) Error:
Rack::Cache::Context#test_0078_passes if there was a metastore exception:
NameError: uninitialized constant Timeout
    test/context_test.rb:981:in `block (3 levels) in <top (required)>'
    test/test_helper.rb:170:in `request'
    test/test_helper.rb:177:in `get'
    test/context_test.rb:980:in `block (2 levels) in <top (required)>'
```

Signed-off-by: Sam James <sam@gentoo.org>
--- a/test/context_test.rb
+++ b/test/context_test.rb
@@ -1,4 +1,5 @@
 require_relative 'test_helper'
+require 'timeout'
 require 'rack/cache/context'
 
 describe Rack::Cache::Context do

